src/main/java/com/cascadebot/cascadebot/utils/FormatUtils.java

```diff
@@ -128,28 +128,28 @@ private static String getFormattedInlineFields(List<MessageEmbed.Field> fieldLis
         List<String> header = new ArrayList<>();
         List<String> body = new ArrayList<>();
 
-        for(MessageEmbed.Field field : fieldList) {
+        for (MessageEmbed.Field field : fieldList) {
             header.add(field.getName());
             body.add(field.getValue());
         }
 
         sb.append('`');
 
-        for(String head : header) {
+        for (String head : header) {
             sb.append(String.format("%-25s", head));
         }
 
-        sb.append("\u200B`");
+        sb.append("\u200B`"); // Zero width space
```

Comment:
In reference to my previous statement having this just be text is fine as zero width space has no content

Reply:
Yeah but immediately looking at this you don't necessarily know it's a zero width space hence the comment

Reply:
Yegh having the text is fine. i was more referring to how we can't really put the unicode character here as zero with space is well 0 width

Reply:
So I was saying ignore my previous review comment in this case

Reply:
Ah sorry for the misunderstanding, yeah I agree